packages/newsroom-signup/src/NewsroomManager/index.tsx

```diff
@@ -0,0 +1,254 @@
+import * as React from "react";
+import { connect, DispatchProp } from "react-redux";
+import { Prompt } from "react-router-dom";
+import styled, { ThemeProvider } from "styled-components";
+
+import { EthAddress, CharterData, NewsroomInstance } from "@joincivil/core";
+import {
+  ButtonTheme,
+  CivilContext,
+  ICivilContext,
+  OBSectionHeader,
+  colors,
+  MetaMaskLogoButton,
+  LoadingMessage,
+  LoadingIndicator,
+  metaMaskLoginImgUrl,
+} from "@joincivil/components";
+
+import { StateWithNewsroom } from "../reducers";
+import {
+  getNewsroom,
+  updateCharter,
+  // analyticsEvent,
```

Comment:
should get rid of the commented out stuff

Reply:
Will do